Wikipedia/UI-V5/WMFArticlePreviewCell.m

```diff
@@ -24,6 +30,18 @@ - (void)prepareForReuse {
     self.summaryLabel.text = nil;
 }
 
+-(void)awakeFromNib {
+    [self rememberSettingsFromIB];
+}
+
+-(void)rememberSettingsFromIB {
+    NSRange r = NSMakeRange(0, 1);
+    self.summaryLabelAttributesFromIB = [self.summaryLabel.attributedText attributesAtIndex:0 effectiveRange:&r];
```

Comment:
Also this wasn't changed either - I think @bgerstle and I are in agreement here that we shouldn't be using IB to get the settings for Attributed Strings. If you want to discus we can, but as I mentioned above its a bit of a hack and I think as Brian as mentioned, it "hides" the configuration of the attributes where no one expects to look.